src/Umbraco.Web.UI.Client/src/views/packages/views/repo.html

```diff
@@ -62,6 +62,12 @@ <h4><strong><localize key="packager_packagesPopular">Popular</localize></strong>
                                             <strong>{{package.likes}}</strong>
                                         </small>
                                     </div>
+                                    <div class="umb-package-cloud">
+                                        <div ng-if="package.certifiedToWorkOnUmbracoCloud">
+                                            <i class="icon-cloud" aria-hidden="true"></i>
```

Comment:
I would suggest using `<umb-icon icon="icon-cloud" class="icon-cloud"></umb-icon>`  ... the class is to work with the font icons, but additional classes can be added.

Reply:
Have updated, thanks.